docs/UserGuide.md

```diff
@@ -102,87 +92,147 @@ There are three main areas in TutorBuddy:
 
 2. the main viewing area,
 
-3. the command box and result display box.
+3. the result display box and command box
```

Comment:
Hi, was just thinking that maybe capitalizing the first letter for each of the sub section would be better. "the result.." to change to "The result". This is the same for 1 and 2 as well.

Reply:
hmmm because I put commas at the end that's why it's left like this.
or should i just remove the commas?

Reply:
I decided to remove the commas and capitalize